src/darwin/mod.rs

```diff
@@ -367,19 +378,25 @@ impl Font {
         let indices = [index as CGGlyph];
 
         unsafe {
-            self.ct_font.get_advances_for_glyphs(
-                kCTFontDefaultOrientation,
-                &indices[0],
+            self.ct_font.advances_for_glyphs(
+                CTFontOrientation::Default,
+                NonNull::from(&indices[0]),
                 ptr::null_mut(),
                 1,
             )
         }
     }
 
     fn get_glyph(&self, character: char, glyph_index: u32) -> RasterizedGlyph {
-        let bounds = self
-            .ct_font
-            .get_bounding_rects_for_glyphs(kCTFontDefaultOrientation, &[glyph_index as CGGlyph]);
+        let glyphs = [glyph_index as CGGlyph];
+        let bounds = unsafe {
+            self.ct_font.bounding_rects_for_glyphs(
+                CTFontOrientation::Default,
+                NonNull::from(&glyphs[0]),
+                ptr::null_mut(),
+                1,
+            )
+        };
```

Comment:
Sorry but what's the point of creating an array just to immediately go and only retrieve its first element?